docs/src/motion/pid-theory.adoc

```diff
@@ -188,4 +188,102 @@ After tuning the axis check the following error with Halscope to make
 sure it is within your machine requirements. More information on
 Halscope is in the HAL User manual.
 
+=== Automatic PID tuning
+
+Since LinuxCNC version 2.9, the pid component support automatic tuning
+using the Relay method footnote:[Åström, Karl Johan and Hägglund, Tore
+(1984), 'Automation paper Automatic Tuning of Simple Regulators with
+Specifications on Phase and Amplitude Margins',
+link:https://doi.org/10.1016/0005-1098(84)90014-1[DOI 10.1016/0005-1098(84)90014-1]].
+This is a replacement for the now removed and obsolete at_pid
+component.
+
+The pid component uses several constants to calculate the output value
+based on current and wanted state, the most important among them being
+Pgain', 'Igain', 'Dgain', 'bias', 'FF0', 'FF1', 'FF2' and 'FF3'. All
+of these need to have a sensible value for the controller to behave
+properly.
+
+The current implementation of automatic tuning implement two different
+algorithms, seleged using the tune-type pin.  When tune-type is zero,
+it affect 'Pgain', 'Igain' and 'Dgain' while setting 'FF0', 'FF1' and
+'FF2' to zero.  If tune-type is 1, it affect 'Pgain', 'Igain' and
+'FF1' while setting 'Dgain', 'FF0' and 'FF2' to zero.  Note type 1
+require scaling be set so output is in user units per second.
+
+When autotuning a motor with tupe-type 0, the algorithm will produce a
+square wave pattern centered around the 'bias' value on the output pin
+of the PID controller, moving from the positive extreme to the
+negative extreme of the output range.  This can be seen using the HAL
+Scope provided by LinuxCNC.  For a motor controller taking +-10 V as
+its control signal, this might accellerate the motor full speed in one
+direction for a short period before telling it to go full speed in the
+opposite direction.  Make sure to have a lot of room on either side of
+the starting position, and start with a low `tune-effort` value to
+limit the speed used.  Using too high 'tune-effort' might overload the
```

Comment:
What is the correlation between tune-effort and controller output signal?
Does tune-effort ? 0.5 limit the output to +-5V in the above example? If yes, would be good to mention.

Reply:
It is one to one.  If tune-effort is set to 0.5, the output will move from 0.5 to -0.5 (ie +-0.5V).  Will try to make it more clear.